apps/stable_diffusion/src/models/__init__.py

```diff
@@ -1,2 +1,10 @@
 from .model_wrappers import SharkifyStableDiffusionModel
-from .opt_params import get_vae, get_unet, get_clip, get_tokenizer
+from .opt_params import (
+    get_vae,
+    get_unet,
+    get_clip,
+    get_tokenizer,
+    get_params,
+    version,
+    variant,
```

Comment:
We should avoid such global variables(as they change with multiple calls) and importing them. As with web, there will be multiple calls with changes to model id's.